sphere-common/src/main/java/io/sphere/sdk/queries/IsGreaterThanOrEqualsQueryPredicate.java

```diff
@@ -0,0 +1,13 @@
+package io.sphere.sdk.queries;
+
+class IsGreaterThanOrEqualsQueryPredicate<T, V, M> extends ComparisonQueryPredicate<T, V, M> {
+
+    public IsGreaterThanOrEqualsQueryPredicate(final QueryModel<M> queryModel, final V value) {
```

Comment:
This class is package scope and my gets removed due to duplication. We can later on discuss together the naming: https://github.com/sphereio/sphere-jvm-sdk/issues/407

But this is not related to this pr.